packages/itwinui-css/src/utils/mixins.scss

```diff
@@ -142,3 +142,9 @@
     @content;
   }
 }
+
+@mixin safari-only {
+  @supports (-apple-pay-button-style: inherit) {
```

Comment:
I have not heard of [`-apple-pay-button-style`](https://css-tricks.com/snippets/css/apple-pay-buttons-in-css/) before.  Just to be sure I understand what is happening here, I'll recap from my perspective:

We're checking if the browser supports `-apple-pay-button-style`.  That only works in Safari, so it's a way of applying styling only for Safari.  We're not actually doing anything with the `-apple-pay-button-style`, we're just using it to target Safari only.

Clever. 🙂